packages/frontend-v2/components/Error/ClientSideError.tsx

```diff
@@ -0,0 +1,12 @@
+export const ClientSideError: React.FC = () => {
+  return (
+    <>
+      <div>
```

Comment:
are we bringing over the styling/other details from the old error page? could also be done later

Reply:
yeap! I forgot to put in a note but i plan to do that once the styling PR gets merged!